test_cromwell.sh

```diff
@@ -112,14 +116,18 @@ fi
 cd centaur
 TEST_STATUS="failed"
 
+sbt test:compile
+CP=$(sbt "export test:dependency-classpath" --error)
+
 if [[ -n ${EXCLUDE_TAG} ]]; then
     echo "Running Centaur filtering out ${EXCLUDE_TAG} tests"
-    TEST_COMMAND="sbt ${REFRESH_TOKEN} \"test-only -- -l ${EXCLUDE_TAG}\""
+    TEST_COMMAND="java ${REFRESH_TOKEN} -cp $CP org.scalatest.tools.Runner -R target/scala-2.11/test-classes -oD -PS${TEST_THREAD_COUNT} -l ${EXCLUDE_TAG}"
```

Comment:
Yeah I pretty much copy pasted this from the `run_tests_parallel.sh` script in centaur.